.github/stale.yml

```diff
@@ -0,0 +1,32 @@
+# Number of days of inactivity before a PR becomes stale.
+daysUntilStale: 60
```

Comment:
i think we agreed on 90 here. other than that, lgtm

Reply:
And we did :) 

If you look at the line bellow, it says "daysUntilClose: 30", so 60 days for a warning + 30 days before closing, and we're at 90 as discussed.